File: Code/Components/DanpheEMR.ServerModel/AccountingModels/ReverseTransactionModel.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace DanpheEMR.ServerModel
{
    public class ReverseTransactionModel
    {

        [Key]
        public int ReverseTransactionId { get; set; }
        public DateTime? TransactionDate { get; set; }
        public int? Section { get; set; }
        public int? TUId { get; set; }
        public int? FiscalYearId { get; set; }
        public string Reason { get; set; }
        public string JsonData { get; set; }
        public DateTime? CreatedOn { get; set; }
        public int? CreatedBy { get; set; }
        //sud-nagesh: 20June'20-- for Acc-Hospital Separation.
        public int? HospitalId { get; set; }
    }
}

File: Code/Websites/DanpheEMR/wwwroot/DanpheApp/src/app/accounting/settings/shared/reverse-transaction.model.ts
import * as moment from "moment";
import { ENUM_DateTimeFormat } from "../../../shared/shared-enums";

export class ReverseTransactionModel {
    public ReverseTransactionId: number = 0;
    public TransactionDate: string = null;
    public Section: number = 0;
    public JsonData: string = null;
    public Reason: string = null;
    public TUId: number = 0;
    public CreatedOn: string = moment().format(ENUM_DateTimeFormat.Year_Month_Day_Hour_Minute);
    public CreatedBy: number = 0;
    public FiscalYearId: number = 0;
}